jest.config.js

```diff
@@ -4,6 +4,7 @@ const { constants } = require('jest-config');
  * @type {import('@jest/types').Config.InitialOptions}
  */
 module.exports = {
+  prettierPath: null,
```

Comment:
Mitigates this error:
```
Jest: Inline Snapshots are not supported when using Prettier 3.0.0 or above.
```

Reply:
Supposedly it is fixed in Jest@30 (alpha) but 29 is still the latest version 😢